tasks/windows_client.yml

```diff
@@ -5,6 +5,10 @@
     #- include_vars: "{{ ansible_distribution }}.yml"
   - include_vars: "{{ ansible_os_family }}.yml"
 
+  - name: Ensure Temp Directory exists
+    win_file:
+      path: "{{ sensu_win_temp_path }}"
```

Comment:
Do you not need `state: directory` for this module?

Reply:
You are right, I glanced over the module and just read the synopsis without realizing I need to set this.